common/app/routes/Challenges/views/invalid/NotFound.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from 'react';
+
+// import PropTypes from 'prop-types';
+/* eslint max-len: ["error", { "ignoreStrings": true }] */
+
+
+const propTypes = {};
+
+export class NotFound extends PureComponent {
+  render() {
+    return (
+        <div className='container'>
+          <div className='row'>
+            <div className='col-sm'>
+            <div className='alert alert-danger' role='alert'>
+              <h4 className='alert-heading' className='mb-12'>
+                This challenge is not found.</h4>
+               <p>Head back to <a className='alert-link'
+                   href='/en/challenges/basic-html-and-html5/learn-how-freecodecamp-works'
```

Comment:
Using the `<Link />` means we don't have to tie this component to a language

Reply:
I am not sure what you meant by "we don't have to tie this component to a language". I used the <Link /> component instead of <a>. However, I kept other styling parts. Is this OK?